include/SKSE/Interfaces.h

```diff
@@ -374,19 +374,26 @@ namespace SKSE
 		constexpr void MinimumRequiredXSEVersion(REL::Version a_version) noexcept { xseMinimum = a_version.pack(); }
 		constexpr void PluginName(std::string_view a_plugin) noexcept { SetCharBuffer(a_plugin, std::span{ pluginName }); }
 		constexpr void PluginVersion(REL::Version a_version) noexcept { pluginVersion = a_version.pack(); }
+		constexpr void HasNoStructUse(bool a_value) noexcept { noStructUse = a_value; }
 		constexpr void UsesAddressLibrary(bool a_value) noexcept { addressLibrary = a_value; }
 		constexpr void UsesSigScanning(bool a_value) noexcept { sigScanning = a_value; }
+		constexpr void UsesStructsPost629(bool a_value) noexcept { structsPost629 = a_value; }
 
 		const std::uint32_t dataVersion{ kVersion };
 		std::uint32_t       pluginVersion = 0;
 		char                pluginName[256] = {};
 		char                author[256] = {};
-		char                supportEmail[256] = {};
-		bool                addressLibrary: 1 = false;
-		bool                sigScanning: 1 = false;
-		std::uint8_t        padding1: 6 = 0;
+		char                supportEmail[252] = {};
+		bool                noStructUse: 1 = false;
+		std::uint8_t        padding1: 7 = 0;
 		std::uint8_t        padding2 = 0;
 		std::uint16_t       padding3 = 0;
+		bool                addressLibrary: 1 = false;
+		bool                sigScanning: 1 = false;
+		bool                structsPost629: 1 = true;
```

Comment:
```suggestion
		bool                structsPost629: 1 = false;
```
This flag implies the plugin works with the pre _and_ post 1.6.629 change, which CommonLib doesn't have support for

Reply:
Isn’t that the other flag? noStructUse? I think this one prevents pre-629 from loading plugins built with this bit set.

Reply:
That said, while I expect nearly every client to have to set this flag, I do get it if we don’t want to be opinionated in proactively setting this for them. Either way is good to me.

Reply:
I was looking at the wrong flag from SKSE, this looks fine to me